release-content/release-notes/ornamented_text2d.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Ornamented `Text2d`"
```

Comment:
This cuts the release note for UI text background colors, only UI text shadows were included in 0.16.